src/controllers/userController.js

```diff
@@ -138,6 +138,24 @@ class Authentication {
       });
     })(req, res);
   }
+
+  /**
+   * @description user logout
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} logged out user
+   */
+  static async signoutUser(req, res) {
```

Comment:
for consistency use logout since to get into the app we are going with login

Reply:
done